.github/workflows/snapshot-build.yml

```diff
@@ -29,12 +29,12 @@ jobs:
           - node
         platform:
           - arch: linux/amd64
-            profile: production
+            dockerfile-suffix: ''
             suffix: ubuntu-x86_64-${{ github.ref_name }}
             image-suffix: ''
-          - arch: linux/arm64
-            # Build at least something for aarch64 that doesn't exceed CI's time limit
-            profile: dev
+          # We build AArch64
+          - arch: linux/amd64
```

Comment:
what is the reason that this is not `arm64` if we are using this for aarch64 builds?

Reply:
was it faster to use amd64 instead of arm64? I remember that cross-compilation was surprisingly fast :D

Reply:
Check the rest of the changes, we use cross-compilation instead of virtualization now